lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -31,6 +35,16 @@
 
                 postRender: function() {
                     // Add subviews
+                    new CollectionListView({
+                        el: '.js-course-list',
+                        childView: CourseCardView,
+                        collection: new Backbone.Collection(this.context.course_codes),
```

Comment:
I debugged into the issue and here is what I found:
If I render instantiate the Collection view before the ProgramDetailsView is rendered, the `this.$el` property of the Collection view would not be able to reference the html div `.js-course-list`. Therefore the render() of the Collection view shows blank. Basically, I need the ProgramDetailsViews to render before the CollectionView is instantiated. for CollectionView to get the correct `this.$el` value.
This means I can either instantiate and render the Collection view in the `initialize()` function after `this.render();` call, or I can instantiate and render the Collection view in the `postRender()`, which is what I have.
I say stick with what I have is better.